types/object-assign-deep/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for object-assign-deep 0.4
+// Project: https://github.com/saikojosh/Object-Assign-Deep#readme
+// Definitions by: Nick Clifford <https://github.com/nickbclifford>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+declare const objectAssignDeep: ObjectConstructor['assign'];
```

Comment:
I'm not sure what are you trying to achieve here. Isn't this a `function` with n parameters or am I missing something?

Reply:
Yes, but it effectively has the same API as `Object.assign`, so I figured that this might be cleaner than rewriting those types. This is my first new definition and I wasn't really sure about the convention, so I went with my gut on that.

Reply:
Makes sense. Thanks for clarifying.